content/en/docs/configuration/external.md

```diff
@@ -5,58 +5,59 @@ weight: 50
-  for cloud native/hybrid environments.
+### Issuers Which Honor Approval
 
+These external issuers are known to support and honor [approval](https://cert-manager.io/docs/concepts/certificaterequest/#approval).
+
+- [kms-issuer](https://github.com/Skyscanner/kms-issuer): Requests
+  certificates signed using an [AWS KMS](https://aws.amazon.com/kms/) asymmetric key.
+- [aws-privateca-issuer](https://github.com/cert-manager/aws-privateca-issuer): Requests
+  certificates from [AWS Private Certificate Authority] (https://aws.amazon.com/certificate-manager/private-certificate-authority/)
+  for cloud native/hybrid environments.
 - [google-cas-issuer](https://github.com/jetstack/google-cas-issuer): Used
   to request certificates signed by private CAs managed by the
   [Google Cloud Certificate Authority Service](https://cloud.google.com/certificate-authority-service/).
-
 - [origin-ca-issuer](https://github.com/cloudflare/origin-ca-issuer): Used
   to request certificates signed by
   [Cloudflare Origin CA](https://developers.cloudflare.com/ssl/origin-configuration/origin-ca)
   to enable TLS between Cloudflare edge and your Kubernetes workloads.
+- [step-issuer](https://github.com/smallstep/step-issuer): Requests
+  certificates from the [Smallstep](https://smallstep.com) [Certificate Authority server](https://github.com/smallstep/certificates).
 
-- [step-issuer](https://github.com/smallstep/step-issuer): Used to request
-  certificates from the [Smallstep](https://smallstep.com) [Certificate
-  Authority server](https://github.com/smallstep/certificates).
+### Issuers Which Don't Honor Approval
 
+{{% pageinfo color="warning" %}}
 
-# Issuers that do NOT Honour Approval
-A list of known external issuer projects that are maintained by their authors
-are as follows. These issuers do _not_ honour
-[approval](../../concepts/certificaterequest/#approval).
+⛔️ These external issuers do _not_ honor [approval](../../concepts/certificaterequest/#approval),
+which means that they might sign a certificate request even if it's been set to "Denied".
 
-- [awskms-issuer](https://github.com/Skyscanner/kms-issuer): Used to request
-  certificates signed using an [AWS KMS](https://aws.amazon.com/kms/) asymmetric key.
+Use these issuers with care if you have workflows which use approval!
 
-- [freeipa-issuer](https://github.com/guilhem/freeipa-issuer): Used to
-  request certificates signed by [FreeIPA](https://www.freeipa.org).
+{{% /pageinfo %}}
 
-- [ADCS Issuer](https://github.com/nokia/adcs-issuer): Used
-  to request certificates signed by 
-  [Microsoft Active Directory Certificate Service](https://docs.microsoft.com/en-us/windows-server/networking/core-network-guide/cncg/server-certs/install-the-certification-authority).
+- [freeipa-issuer](https://github.com/guilhem/freeipa-issuer): Requests
+  certificates signed by [FreeIPA](https://www.freeipa.org).
+- [ADCS Issuer](https://github.com/nokia/adcs-issuer): Requests
```

Comment:
Likewise https://github.com/nokia/adcs-issuer/pull/10, but the project is looking for maintainers.